### ISSUE TITLE
Maximum resolution

### ISSUE THREAD

**Issue by Jammy1986** [fnordware/AdobeWebM#37]
Hi, I have been looking into codecs which can support resolutions greater than 32000, and ended up finding VP9, and this plugin. I found a line in "WebM_Premiere_Export_Params.cpp" which limits the resolution to 8192, and I was wondering if this is an arbitrary limitation which could be raised, or whether that value was chosen for a reason?

Unfortunately my C++ is very rusty and I couldn't find any instructions for compilation. The only thing I did try was to specify a -w 16000 argument in the options box, but then noticed that the export params file doesn't handle that argument (which is fair enough). 

I guess in summary, is it possible to increase this maximum value?

Thanks

**Comment by fnordware** [comment]
I guess it's a little arbitrary, although I think Premiere forces me to have *some* maximum value.  Can you actually move a 32,000 pixel movie through Premiere or Media Encoder?

I could make you a build with bigger limits if you want to try it out. Email me: brendan at fnordware dot com.

**Comment by nikolaymatveev** [comment]
It's not my case, but what means "Can you actually move a 32,000 pixel movie through Premiere or Media Encoder"? To process such resolution while processing and applying effects or to use some other exporter - i.e. ability to save?

**Comment by Jammy1986** [comment]
Yeah, I'm pretty sure he was referring to whether AME would run out of memory while encoding. It doesn't, and encodes fine for my purposes. :)

**Comment by nikolaymatveev** [comment]
Right now I encoding 2-hours 720p@30fps video to VP9 VBR 1500kbps 2-pass (color sampling as is 4:2:0, depth as as 8 bits) via A.Premiere 2017. Analyze phase ends with 13.08 GB RAM usage (it is 10% threshold on progress line) and for now it is 18% progress with 14.07 GB RAM usage. So utilizing RAM size is still increasing. BTW I have to turn on pagefile.sys usage back, cause recent crash of WebM exporter plug-in - it was not enough memory (15.7 GB available). 

What is your experience on time to encode and memory usage on each type of video while encoding in VPx?

**Comment by fnordware** [comment]
Yep, Jamey is right, I wasn't sure Media Encoder could handle it. I guess it can! He figured a good max resolution would be 16384, so I'll set that in an upcoming build.
